Server/src/services/dataExtraction/DataExtractionService.ts

```diff
@@ -28,9 +29,19 @@ export class DataExtractionService {
             //this.factory = new PDFFileExtractorFactory()
         }
         let fileHandler: any = await this.factory.getFileHandler(this.filePath)
-        let parsedData = await fileHandler.parseFile()
-        requestResponse.message = parsedData
-        requestResponse.statusCode = 200;
-        return requestResponse;
+        try {
+            let parsedData = await fileHandler.parseFile()
+            requestResponse.message = parsedData
+            requestResponse.statusCode = 200;
+            return requestResponse;
+        }
+        catch (error) {
```

Comment:
why the branching in the errors?

Reply:
Is there a reason we wouldnt want the branching?